src/main/java/com/olgaboiar/mint/Server.java

```diff
@@ -34,8 +33,8 @@ public void run() throws IOException {
         acceptClientConnection();
         listenToClientConnection();
         readClientInput();
-        String[] responseArray = responseGenerator.generateResponse();
-        sendResponseToClient(responseArray);
+//        Response response = responseGenerator.generateResponse();
```

Comment:
Remove the commented out code.

Reply:
removed in https://github.com/olgaboiar/MintHTTPServer/pull/6/commits/39acfd8c01a36590f549c1985adc08a5dcbf86bf